client/src/pages/EditProfile/useStyles.ts

```diff
@@ -0,0 +1,52 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%',
+    padding: '0 40px 70px',
+  },
+  typo: {
```

Comment:
maybe `typo` is not the best variable, as it's like type mistake 😅  i suggest maybe "typography" or something more meaningful

Reply:
Resolved